docs/docs/guides/compatibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: compatibility
+title: Compatibility
```

Comment:
```suggestion
```

The `title: Compatibility` in the front matter and `# Compatibility Table` heading contradict with each other. Using both doesn't make sense as the second one overrides the first one.